src/main/kotlin/app/ui/userCreation/Model/UserCreationModel.kt

```diff
@@ -0,0 +1,24 @@
+package app.ui.userCreation.Model
+
+import java.awt.Image
+
+class UserCreationModel {
+
+    var isRecording = false
+    lateinit var audioFile: String
+    lateinit var UserIcon: Image
```

Comment:
userIcon

also, a nullable type is probably more appropriate here than a lateinit.